src/domain/consents/index.ts

```diff
@@ -0,0 +1,25 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation The Mojaloop files are made available by the Mojaloop Foundation
+ under the Apache License, Version 2.0 (the 'License') and you may not
+ use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in
+ writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ OF ANY KIND, either express or implied. See the License for the specific language governing
+ permissions and limitations under the License. Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file. Names of the original
+ copyright holders (individuals or organizations) should be listed with a '*' in the first column.
+ People who have contributed from an organization can be listed under the organization that actually
+ holds the copyright for their contributions (see the Gates Foundation organization for an example).
+ Those individuals should have their names indented and be marked with a '-'. Email address can be
+ added optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+
+ --------------
+ ******/
+export * as ConsentsId from './{ID}'
```

Comment:
why the file name `"{ID}.ts"`? That looks to me like the notions of the http layer influencing the domain, which is the wrong way around.

Reply:
seemed like a logical way to structure the file directory. I'm unsure of the preferred way to structure this.

Reply:
It's just that naming a file `{ID}.ts` is an artifact of the hapi library we were using, and has nothing to do with the business logic that lives in `./domain/`.

Why not just move the `forwardConsentsIdRequestError` function in with `./domain/consents.ts`

Reply:
done. though I'm regretting it a little bit since the diff is hard to read now.
flattened domain structure for consents and consentRequests for consistency.